docs/lts_policy.rst

```diff
@@ -2,20 +2,6 @@
 LTS Backport Policy
 *******************
 
-Starting from astropy 5.0, backports to Long-Term Stable (LTS) releases
-will only cover:
-
-* Critical security fixes.
-* Bug fixes where backporting is straightforward (e.g., no conflicts).
-* More complex fixes are permissible only as resources allow, ideally with
-  backports done by the original author, users, or institutions who are on LTS
-  and need the fix.
-* New "features" that are absolutely necessary to accomplish the above.
-* Other critical additions absolutely necessary for users stuck to the LTS.
-  In this case, the users themselves (e.g., developers from affected institutions)
-  would perform the needed backports or implementation.
-
-This is because LTS lasts for about 2 years. During that time frame, as the
-LTS branch diverges from the development branch, backports will become
-increasingly difficult due to merge conflicts. When conflicts arise,
-automation is not possible, therefore driving up the maintenance cost.
+Starting with astropy v6.0.0, there will be no more designated Long-Term Stable
+(LTS) releases of astropy - see `APE 21
+<https://github.com/astropy/astropy-APEs/blob/main/APE21.rst>`_ for more details.
```

Comment:
Now that this file only say "we don't have LTS any more" and it's only three lines long, should the entire file be removed and the info just one one level higher?

Reply:
Fair enough.